tuna.go

```diff
@@ -560,3 +562,44 @@ func LoadOrCreateAccount(walletFile, passwordFile string) (*vault.Account, error
 	}
 	return wallet.GetDefaultAccount()
 }
+
+func sendNanoPayment(np *nkn.NanoPay, session *smux.Session, w *nkn.Wallet, inBytes, outBytes, inBytesPaid, outBytesPaid *uint64, c *Common, nanoPayFee string) {
```

Comment:
`sendNanoPayment` is a function shared by entry and exit. As I said earlier, `in` and `out` have different meaning on entry and exit side: at entry, `in` means exit to entry, while at exit, `in` means entry to exit. So we need to change all `in` and `out` related variables in this function to `entryToExitXXX` and `exitToEntryXXX`.

Reply:
Also it's better for this function to return error, so the caller can decide what to do